frontend/src/components/auth/Signup.tsx

```diff
@@ -82,7 +88,7 @@ const Signup = (): React.ReactElement => {
             </Center>
             <FormControl
```

Comment:
Could you change each of the form elements (label + input) to be in their own form control, instead of all being in the same one? I think this will fix some of the autofill issues we're experiencing still on this page.